sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/flink-functions/collections-functions.sqrl

```diff
@@ -0,0 +1,59 @@
+IMPORT flink.*; -- Import flink functions
+IMPORT data-local.Arrays; -- Import array data
+
+/* Create numeric data source */
+UnmodifiedArrayData := SELECT id,
+                              timestamp,
+                              items
+                       FROM Arrays ORDER BY id;
+
+
+/* =======TEST CASES======== */
+
+/*+test */
+CardinalityTest := SELECT id,
+                          CARDINALITY(items) as cardinality,
+                          items[1] as array_at_int,
+                          ELEMENT(ARRAY[1]) as element_sole_value
+                          /*ELEMENT(CAST(ARRAY[NULL] AS VARCHAR)) as element_empty*/ -- can't declare empty array
+                   FROM UnmodifiedArrayData
+                   ORDER BY id;
+
+/*+test */
+ArrayTest := SELECT id,
```

Comment:
This should work. The commented out ones don't exist in our version of flink. Also I did a few spelling changes.
```
ArrayTest := SELECT id,
                    /*ARRAY_APPEND(ARRAY[1, 2, 3], 4) as append_array,*/
                    ARRAY_CONTAINS(ARRAY['1','2','3','4'], '3') as array_contains,
                    ARRAY_DISTINCT(ARRAY['1','1','2','2']) as array_distinct,
                    ARRAY_POSITION(ARRAY['1','2','3','4'], '3') as array_position,
                    /*ARRAY_PREPEND(items, 3) as array_prepend,*/
                    /*ARRAY_REMOVE(ARRAY['1','2','3','4'], ARRAY['3']) as array_remove,*/
                    ARRAY_REVERSE(items) as array_reverse,
                    ARRAY_SLICE(items,0,2) as array_slice,
                    /*ARRAY_SORT(items) as array_sort,*/
                    ARRAY_UNION(ARRAY['1','2','3','4'], ARRAY['3','4','5','6']) as array_union,
                    ARRAY_CONCAT(ARRAY['1','2','3'], ARRAY['4','5','6']) as array_concat,
                    /*ARRAY_EXCEPT(ARRAY[1,2,3], ARRAY[4,5,6]) as array_except,*/
                    /*ARRAY_INTERSECT(ARRAY[1,2,3], ARRAY[4,5,6]) as array_intersect,*/
                    ARRAY_MAX(ARRAY[1,2,3]) as array_max,
                    ARRAY_JOIN(ARRAY['1','2','3'], ';') as array_join,
                    ARRAY_MIN(ARRAY[1,2,3]) as array_min
             FROM UnmodifiedArrayData
             ORDER BY id;
```